go.mod

```diff
@@ -11,6 +11,7 @@ require (
 	github.com/sirupsen/logrus v1.5.0
 	github.com/spf13/pflag v1.0.5
 	github.com/stretchr/testify v1.6.1
+	golang.org/x/tools v0.1.0 // indirect
```

Comment:
I might be missing something but why are you adding this?

Reply:
good question, it was done automatically when I did make code/gen. I will revert that